content/200-concepts/300-more/050-environment-variables/index.mdx

```diff
@@ -16,29 +16,6 @@ You can use machine-specific variables, `.env` files, or a combination of both.
 
 </TopBlock>
 
-## Accessing environment variables from the schema
-
-Environment variables can be accessed using the `env` function:
-
-```prisma
-datasource db {
-  provider = "postgresql"
-  url      = env("DATABASE_URL")
-}
-```
-
-In version 2.7.0 and later, you can expand variables using the format specified by [dotenv-expand](https://github.com/motdotla/dotenv-expand). This is useful when dealing with PaaS or similar products. such as Heroku):
-
-```
-# .env
-DATABASE_URL=postgresql://test:test@localhost:5432/test
-DATABASE_URL_WITH_SCHEMA=${DATABASE_URL}?schema=public
-```
-
-There are a few limitations with `env` at the moment:
-
-- It is not possible to use environment variables for the `provider` argument in `datasource` and `generator` definitions.
-
 ## Using .env files
 
 For many developer tools, it has become a good practice to define environment variables using [`.env`](https://github.com/motdotla/dotenv) files. The Prisma CLI looks for `.env` files, in order, in the following locations:
```

Comment:
We probably need a section about Prisma Client's use of `.env` as well then. Not sure if this is identical, then it should be added here as well.